examples/providerconfig/env.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: gcp.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: example
+spec:
+  projectID: PROJECT_ID
+  credentials:
+    source: Filesystem
+    fs:
+      path: ./crossplane-gcp-provider-key.json
```

Comment:
Is it worth clarifying in the example what this path is relative to? (Presumably the root of the container?)

Reply:
@negz sure :+1: it would be relative to process wd